.github/goodtables_report.py

```diff
@@ -43,8 +46,43 @@
 ERROR_FILE = os.environ['ERROR_FILE']
 REPO = os.environ['GITHUB_REPOSITORY']
 ERROR_MAX = int(os.environ['ERROR_MAX'])
+SCHEMA_PATH = os.environ['SCHEMA_PATH']
+
+def is_invalid_schema():
+    """Send a Slack message and return true if the datapackage json is invalid."""
+
+    try:
+        with open(os.environ['SCHEMA_PATH']) as f:
+            json.load(f)
+    except Exception as err:
+        j_payload = {
+            'text': "There was an error loading the JSON:",
+            'attachments': [
+                {
+                    "fallback": f'```{str(err)}```',
+                    "color": "danger",
+                    "fields":[{
+                       "title": os.environ['SCHEMA_PATH'],
+                       "value": f'```{str(err)}```'
+                    }]
+                }
+            ]
+        }
+
+        # post stringified payload to url
+        requests.post(os.environ['SLACK_GOODTABLES_WEBHOOK'], json.dumps(j_payload))
```

Comment:
Would be a little cleaner and simpler to get the environment variables once like you do the other global variables. Could even make a list of slack webhooks to simplify the logic where you're checking and posting twice — although I think you only do it twice, so not a huge gain.